.github/workflows/ci.yml

```diff
@@ -283,7 +288,7 @@ jobs:
           SDK_COMMANDS_S3_URL: ${{ secrets.SDK_TEAM_S3_BASE_URL }}/${{ steps.publish_decentraland_sdk_commands.outputs.s3-bucket-key }}
 
       - name: publish @dcl/sdk package
-        uses: menduz/oddish-action@master
+        uses: nicoecheza/oddish-action@patch-1
```

Comment:
should we move this into the `decentraland` org?

Reply:
that was a test for checking if builds were fixed. Already removed that commit since https://github.com/menduz/oddish-action/pull/23 got merged👌